client/db/types.go

```diff
@@ -71,17 +74,26 @@ type AccountInfo struct {
 	FeeCoin   []byte
 	// Paid will be set on retrieval based on whether there is an AccountProof
 	// set.
-	Paid bool
+	Paid         bool
+	ClientPubKey *secp256k1.PublicKey
+	ID           account.AccountID
+	Disabled     bool
```

Comment:
Grouping the two `bool`s saves allows a more compact struct in memory with less padding.